### DISCUSSION TITLE
Weaviate

### DISCUSSION THREAD

**Discussion by AruneshSingh** [superlinked/VectorHub#67] [VDB Comparison]
Hey everyone, this is the start of discussions related to [Weaviate](http://weaviate.io/) and their attributes as compared in the table [here](https://vdbs.superlinked.com). 

To add to the discussion please check if there's already a comment with the attribute you want to discuss about, and reply in the thread. Otherwise please create a new comment for the attribute you want to discuss about.

To create an issue, please have a look at the [README](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) for contributing guidelines.

Let's all try to have a fruitful discussion!

**Comment by Dabz** [comment]
Hi @AruneshSingh,

Looking quickly at the table, Weaviate does support Scalar & 8-bit rotational quantization. Source:

- https://docs.weaviate.io/weaviate/configuration/compression/sq-compression
- https://docs.weaviate.io/weaviate/configuration/compression/rq-compression

As well, I do understand your comment regarding Binary Quantization: `via Binary Quantization of index` - but I find the color (orange/yellow) a bit misleading. Weaviate perform a Binary Quantization of the index to store both compressed & uncompressed vectors, it performs automatically a rescoring on the uncompressed vector to improve the accuracy of the queries.
